flojoy/utils.py

```diff
@@ -20,7 +21,11 @@
 env_vars = dotenv_values("../.env")
 port = env_vars.get("VITE_BACKEND_PORT", "8000")
 BACKEND_URL = os.environ.get("BACKEND_URL", f"http://127.0.0.1:{port}")
-is_offline = False
+
+
+def flojoy_print(*args, **kwargs):
```

Comment:
Is there a reason for not using a debugger and specifically configure the log level?

Reply:
definitely overkill, no reason to have a debugger inside the package currently since all logs are debug logs

Reply:
ok switched to logger instead